batch/batch/front_end/templates/billing_projects.html

```diff
@@ -3,57 +3,88 @@
 {% block content %}
 <h1>Billing Projects</h1>
 <div class="searchbar-table">
-  <table class="data-table" id="batches">
+  <table class="grouped-data-table" id="billing-projects">
     <thead>
       <tr>
-        <th>Billing Project</th>
-        <th>User</th>
-        <th></th>
+        <th colspan="2">Billing Project</th>
+        <th colspan="2">User</th>
       </tr>
     </thead>
-    <tbody>
       {% for billing_project in billing_projects %}
+    <tbody>
```

Comment:
Why is the tbody under the for loop now?

Reply:
Each billing project is now wrapped in its own `<tbody>` tag.

The big UI change I made here were to treat the billing project (and the "close" button) as a multi-row cell that spans the height of its users, instead of having it as a distinct row in front of the users. I prefer this because the visual grouping is cleaner, but I can switch it back if you'd prefer.

What this meant was that there was a weird color thing going on with the billing project cell depending on whether the billing project had an even or odd number of users (since they're colored according to the color of the top row, since they technically "live" there). 

I solved this by changing the color pattern to be alternate for every other *billing project*, but each user still needed to have its own row. This meant wrapping each billing project in a `<tbody>` tag and defining a new type of table that has the `:nth-of-type(even)` style definition on the `tbody` tab instead of the `tr` tag.

This means that there's no color differentiation between users in a row, but the table is short enough horizontally that I think I prefer this to the other way (visually differentiated users but not billing projects). The on-hover background change still applies to row, so it's still pretty obvious which user you're trying to remove.